Sources/Valet/SecureEnclaveAccessControl.swift

```diff
@@ -97,18 +97,16 @@ public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable
     }
 
     internal static func allValues() -> [SecureEnclaveAccessControl] {
```

Comment:
There are no changes to the output of this method. However when investigating adding more cases I found that the previous setup of this method made it hard to add new cases with different `#available` configurations, so I restructured this method to be a bit easier to maintain over time.